rebalance/rebalance.py

```diff
@@ -546,8 +612,25 @@ def init(options, configuration, plugin):
     plugin.fee_base = Millisatoshi(config.get("fee-base"))
     plugin.fee_ppm = config.get("fee-per-satoshi")
     plugin.mutex = Lock()
-    plugin.log(f"Plugin rebalance initialized with {plugin.fee_base} base / {plugin.fee_ppm} ppm fee, "
-               f"cltv_final: {plugin.cltv_final}")
-
-
+    plugin.maxhops = int(options.get("rebalance-maxhops"))
+    plugin.msat_factor = float(options.get("rebalance-msat_factor"))
+    plugin.log(f"Plugin rebalance initialized with {plugin.fee_base} base / {plugin.fee_ppm} ppm fee  "
+               f"cltv_final:{plugin.cltv_final}  "
+               f"maxhops:{plugin.maxhops}  "
+               f"msat_factor:{plugin.msat_factor}")
+
+plugin.add_option(
+    "rebalance-maxhops",
+    "6",
+    "Maximum number of hops for `getroute` call. "
```

Comment:
This is the starting value of the `maxhops` iteration. Maybe you intended to set it to "1"?

Reply:
lol, thx

Reply:
fixed in recent force push